src/System.Text.Json/src/System/Text/Json/Serialization/JsonSerializerOptions.cs

```diff
@@ -41,29 +57,24 @@ public int DefaultBufferSize
             }
             set
             {
-                if (value == 0 || value < BufferSizeUnspecified)
+                if (value < 1)
                 {
                     throw new ArgumentException(SR.SerializationInvalidBufferSize);
                 }
 
                 _defaultBufferSize = value;
-
-                if (_defaultBufferSize == BufferSizeUnspecified)
-                {
-                    EffectiveBufferSize = BufferSizeDefault;
-                }
-                else
-                {
-                    EffectiveBufferSize = _defaultBufferSize;
-                }
             }
         }
 
+        /// <summary>
+        /// Determines whether null properties are included when serializing.
```

Comment:
And if they are (i.e. true), what does included mean? `Null`, `""`, etc.

Reply:
Changed to `Determines whether null values of properties are ignored or whether they are written to the JSON.`